home.html

```diff
@@ -0,0 +1,201 @@
+              sharing and opening
+            </p>
+          </li>
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/exhibition.svg"
+              alt="Icon presentation"
+            />
+            <h6>Exhibition</h6>
+            <p class="paragraph-program">
+              Listen to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/discussion.svg"
+              alt="Icon presentation"
+            />
+            <h6>Forum</h6>
+            <p class="paragraph-program">
+              Liste to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/workshop.svg"
+              alt="Icon presentation"
+            />
+            <h6>Workshop</h6>
+            <p class="paragraph-program">
+              Liste to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/dharma-wheel.svg"
+              alt="Icon presentation"
+            />
+            <h6>Dharma</h6>
+            <p class="paragraph-program">
+              Liste to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+        </ul>
+        <div class="button-program">
+          <button type="button" class="btn-btn">Join Estudio Polanco</button>
+        </div>
+        <div class="link-program">
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+      <section class="models">
+        <div class="title-models">
+          <h4>Featured Models</h4>
+        </div>
+        <hr />
+        <ul class="models-list"></ul>
+      </section>
+      <div class="button-more">
+        <button type="button">MORE &#5167;</button>
+      </div>
+      <div class="desktop-logo">
+        <div class="title-about-four">
+          <h5>Partners</h5>
+          <hr />
+        </div>
+        <div class="logos">
+          <img src="./img/Apple_logo.svg" alt="Logo" class="img-logos" />
+          <img
+            src="./img/Google_Capital_logo.svg"
+            alt="Logo"
+            class="img-logos"
+          />
+          <img src="./img/amazon.svg" alt="Logo" class="img-logos" />
+          <img src="./img/Gsuite_logo.svg" alt="Logo" class="img-logos" />
+          <img src="./img/Pixel_wordmark.svg" alt="Logo" class="img-logos" />
+        </div>
+      </div>
+      <footer>
```

Comment:
- [ ] You should place the logo and the copyright text as same as given in the design template. Currently, Items not placed and aligned correctly.

Screenshot :arrow_down_small: 
![Screenshot from 2021-07-03 00-58-14](https://user-images.githubusercontent.com/1642014/124336172-ab012d80-dba5-11eb-9cfc-2d0be8e89981.png)